### PR TITLE
Update workflow to not break when nothing to commit

### PR DESCRIPTION
**Pull request by harshau007**
## Description

Now workflow won't break when there's nothing to commit as it does. We are using conditional statements for checking if there is any changes in the any file and if there is it will update shasum then push and vice versa 

## Screenshots (if applicable)

N/A

## Checklist
- [x] I have read and agree to abide by the [Code of Conduct](https://github.com/tcet-opensource/.github/blob/main/CODE_OF_CONDUCT.md)
- [x] I have read and followed the **Contributing Guidelines**

### CODE REVIEW

**Comment by Akash6222** [comment]
Update pkgbuild repo url 🙂

**Comment by harshau007** [comment]
> Update pkgbuild repo url 🙂

Done

**Review by Rishabh672003 — CHANGES_REQUESTED** [review]
change the pkgbuild url

**Comment by Rishabh672003** [comment]
wait you have done the changes isk why its not showing in the main `Files changed` section

**Review by Rishabh672003 — APPROVED** [review]

File: .github/workflows/shasum.yml [hunk 1]
```diff
@@ -39,12 +39,28 @@ jobs:
...
+      - name: create pull request
+        if: env.changed_files != ''
+        run: |
+          cd tcet-linux-pkgbuild/
           git add .
-          git remote set-url origin https://x-access-token:${{ secrets.GITHUB_TOKEN }}@github.com/${{ github.repository }}
-          git commit -am "Updated Shasum"
-          git push -u origin main
\ No newline at end of file
+          git remote set-url origin https://x-access-token:${{ secrets.GITHUB_TOKEN }}@github.com/harshau007/tcet-linux-pkgbuild.git
```
Comment (Akash6222) [inline .github/workflows/shasum.yml]:
```suggestion
          git remote set-url origin https://x-access-token:${{ secrets.GITHUB_TOKEN }}@github.com/tcet-opensource/tcet-linux-pkgbuild.git
```

**Review by Akash6222 — APPROVED** [review]
GTG

**Pull request merged**